### PR TITLE
define node neighbor cardinalities from InFactsImporter (2/2)

### PR DESCRIPTION
**Pull request by mpollmeier**
```
TYPE_DECL has 0 to 1 incoming EdgeTypes.AST from NodeTypes.METHOD, 
TYPE_DECL has 0 to 1 incoming EdgeTypes.AST from NodeTypes.TYPE_DECL, 
TYPE_DECL has 0 to 1 incoming EdgeTypes.AST from NodeTypes.NAMESPACE_BLOCK, 
METHOD has 0 to 1 incoming EdgeTypes.AST from NodeTypes.METHOD, METHOD has
0 to 1 incoming EdgeTypes.AST from NodeTypes.TYPE_DECL, METHOD has 0 to 1
incoming EdgeTypes.AST from NodeTypes.NAMESPACE_BLOCK, METHOD_PARAMETER_IN
has 1 incoming EdgeTypes.AST from METHOD METHOD_RETURN has 1 incoming
EdgeTypes.AST from METHOD TYPE_PARAMETER has 1 incoming EdgeTypes.AST from
METHOD or NodeTypes.TYPE_DECL MODIFIER has 1 incoming EdgeTypes.AST from
METHOD or NodeTypes.METHOD_PARAMETER_IN! or NodeTypes.TYPE_DECL or
NodeTypes.MEMBER! MEMBER has 1 incoming EdgeTypes.AST from TYPE_DECL 
NAMESPACE_BLOCK has 0 to 1 incoming EdgeTypes.AST from FILE LITERAL has 1
incoming EdgeTypes.AST from Expression or NodeTypes.CONTROL_STRUCTURE 
LITERAL has 0 to 1 incoming EdgeTypes.ARGUMENT from CALL or
NodeTypes.RETURN CALL has 1 incoming EdgeTypes.AST from Expression or
NodeTypes.CONTROL_STRUCTURE CALL has 0 to 1 incoming EdgeTypes.ARGUMENT
from CALL or NodeTypes.RETURN LOCAL has 1 incoming EdgeTypes.AST from BLOCK 
IDENTIFIER has 0 to 1 incoming EdgeTypes.ARGUMENT from CALL or
NodeTypes.RETURN IDENTIFIER has 1 incoming EdgeTypes.AST from Expression or
NodeTypes.CONTROL_STRUCTURE RETURN has 1 incoming EdgeTypes.AST from
Expression or NodeTypes.CONTROL_STRUCTURE BLOCK has 1 incoming
EdgeTypes.AST from Expression or NodeTypes.METHOD or
NodeTypes.CONTROL_STRUCTURE BLOCK has 0 to 1 incoming EdgeTypes.ARGUMENT
from CALL or NodeTypes.RETURN METHOD_REF has 0 to 1 incoming
EdgeTypes.ARGUMENT from CALL or NodeTypes.RETURN FIELD_IDENTIFIER has 1
incoming EdgeTypes.AST from CALL FIELD_IDENTIFIER has 1 incoming
EdgeTypes.ARGUMENT from CALL METHOD_REF has 1 incoming EdgeTypes.AST from
Expression or NodeTypes.CONTROL_STRUCTURE BINDING has 1 incoming
EdgeTypes.BINDS from TYPE_DECL

### CODE REVIEW

File: codepropertygraph/src/main/resources/schemas/base.json [hunk 1]
```diff
@@ -89,10 +89,10 @@
            {"edgeName": "AST",
             "inNodes": [
               {"name": "METHOD_RETURN", "cardinality":"1:1"},
-              {"name": "METHOD_PARAMETER_IN"},
-              {"name": "MODIFIER"},
+              {"name": "METHOD_PARAMETER_IN", "cardinaliy": "n:1"},
```
Comment (ml86) [inline codepropertygraph/src/main/resources/schemas/base.json:92]: I think you mixed up the ordering:
A METHOD can connect via AST to n METHOD_PARAMETR_IN
and
A METHOD_PARAMETER_IN can have 1 incoming AST from METHOD.

But you are stating it the other way around. Look at the IDENTIFIER to LOCAL via REF relation as a correct example.

Comment (mpollmeier) [reply codepropertygraph/src/main/resources/schemas/base.json:92]: thanks for paying attention. 
maybe this was just a test? you passed it either way. 
will review all and let you know

Comment (mpollmeier) [reply codepropertygraph/src/main/resources/schemas/base.json:92]: yup, they all need to be reversed...

**Comment by mpollmeier** [comment]
ready for re-review @ml86

**Pull request merged**